src/templates/ObservabilityPackDetails.js

```diff
@@ -24,9 +25,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
               gap: 1rem;
             `}
           >
-            <Button variant={Button.VARIANT.PRIMARY} size={Button.SIZE.MEDIUM}>
-              Install
-            </Button>
+            <InstallButton
+              title="Install"
+              guid={pack.id}
+              onClick={() => {
+                tessen.track('observabilityPack', `ObservabilityPackInstall`, {
+                  installPackName: pack.name,
+                  installPackId: pack.id,
+                });
+                if (typeof window !== 'undefined' && window.newrelic) {
+                  window.newrelic.addPageAction('oPackInstall', {
```

Comment:
I think we can take advantage of [`useInstrumentedHandler`](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#useinstrumentedhandler) here (to avoid doing extra checks for `window` and abstracting away the `addPageAction` stuff.

Reply:
Ye I wasn't 100% sure how to use that within this context, maybe we can chat about this Monday too :D

Reply:
I _think_ we can call `useInstrumentedHandler` to get the callback function used for the click event. Not 100% sure if this works, but here's the relevant bits of code I was thinking:

```jsx
// import the two hooks

import { useTessen, useInstrumentedHandler } from '@newrelic/gatsby-theme-newrelic';

// call the two hooks before the JSX code

const installPackData = { installPackName: pack.name, installPackId: pack.id };

const tessen = useTessen();

const handleInstallClick = useInstrumentedHandler(
  () => {
    tessen.track('observabilityPack', 'ObservabilityPackInstall', installPackData);
  },
  installPackData,
  'oPackInstall'
);

// use the handler from the hook in the JSX

<InstallButton
  onClick={handleInstallClick}
/>
```